app/store/localStorage.js

```diff
@@ -16,8 +19,18 @@ function setItem(key, value) {
 }
 
 function getItem(key) {
-  return (isBrowser && window.localStorage &&
-    window.localStorage.getItem(key)) || null;
+  if (isBrowser && window.localStorage) {
+    try {
+      return window.localStorage.getItem(key);
+    } catch (error) {
+      if (error.name === 'SecurityError') {
```

Comment:
Could the catch block (or even the whole try/catch) be extracted, so it doesn't need to be duplicated?